### DISCUSSION TITLE
Domain limit with Vercel platforms

### DISCUSSION THREAD

**Discussion by AnishDe12020** [vercel/community#437] [Help] [answered]
So currently, there is a limit of 50 domains per project on a vercel project. Now, if I set up Vercel platforms and provide my users with an option to claim a subdomain, will I be limited to maybe just 48 users being able to claim a subdomain (given that I have to assign some for my applications), or is it unlimited?

Also, if I want to give them the option to set up a custom domain, how does the process work? Is it like they add a domain that gets associated with my account or something? Do I need to set up anything extra for it or does Vercel handle it all? Also, what are the limits? (taking into consideration it will be running on the hobby plan)

**Comment by Pieparker** [comment, marked as answer]
[This process](https://vercel.com/guides/nextjs-multi-tenant-application#4.-configure-rewrites-for-multi-tenancy) is going to be useful for you! Rewrites using edge functions have you covered.
